docs/pages/kcardgrid.vue

```diff
@@ -1,907 +1,950 @@
-<template>
```

Comment:
This is a huge diff which should not be present. This is probably due to some local formatting tool? Could you first resolve the linting check fails and then run `yarn lint-fix`.